packages/react-charting/src/components/SankeyChart/SankeyChart.types.ts

```diff
@@ -113,3 +118,29 @@ export interface ISankeyChartStyles {
 
   calloutContentRoot?: IStyle;
 }
+
+/**
+ * Set of strings which are used when renering the chart.
+ */
+export interface ISankeyChartStrings {
+  /**
+   * Aria label for when the chart is empty.
+   */
+  emptyAriaLabel: string;
```

Comment:
>string [](http://example.com/codeflow?start=18&length=6)

Use IAccessibilityProps interface instead of string and keep the default english value for backward compatibility